packages/asc-ui/src/components/InfoBanner/InfoBanner.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Close } from '@datapunt/asc-assets'
+import InfoBannerStyle, {
+  Props,
+  CloseButton,
+  CloseButtonWrapper,
+  ContentWrapper,
+} from './InfoBannerStyle'
+import Heading from '../Heading'
+import CompactThemeProvider from '../../theme/CompactThemeProvider'
+import Paragraph from '../Paragraph'
+import Link from '../Link'
+import { ButtonVariants } from '../Button/ButtonStyle'
+
+const CLOSE_BUTTON_TITLE = 'Sluiten'
+
+const InfoBanner: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
+  children,
+  heading,
+  onDismiss,
+  dismissible,
+  compact,
+  content,
+  level,
+  linkProps,
+  ...otherProps
+}) => {
+  const [open, setOpen] = useState(true)
+
+  let variant: keyof typeof ButtonVariants = 'tertiary'
+
+  if (level === 'error') {
+    variant = 'secondary'
+  } else if (level === 'attention') {
+    variant = 'primary'
+  }
+
+  const handleOnDismiss = () => {
+    setOpen(false)
+
+    if (onDismiss) {
+      onDismiss()
+    }
+  }
+
+  return open ? (
+    <InfoBannerStyle
+      {...{ compact, dismissible, level, ...otherProps }}
+      tabIndex={0}
+      role="alert"
+      aria-live="polite"
+    >
+      <CompactThemeProvider>
+        <ContentWrapper compact={compact}>
+          {heading && (
+            <Heading forwardedAs="strong" styleAs={compact ? 'h5' : 'h3'}>
+              {heading}
+            </Heading>
+          )}
+          {content && <Paragraph>{content}</Paragraph>}
+          {linkProps && (
+            <Link
+              onDarkBackground={level !== 'normal'}
+              title={linkProps.title}
+              href={linkProps.href}
+              variant="with-chevron"
+            >
+              {linkProps.title}
+            </Link>
+          )}
+          {children}
+        </ContentWrapper>
+        {dismissible && (
+          <CloseButtonWrapper compact={compact}>
+            <CloseButton
+              title={CLOSE_BUTTON_TITLE}
+              aria-label={CLOSE_BUTTON_TITLE}
+              size={compact ? 24 : 30}
```

Comment:
Should we use a multiplication of the theme spacing here?

Reply:
For now we only use themespacing for margins and paddings, not really for height and width now. Of course we can discuss how we can also use it for these dimensions but that would be out of scope for this ticket